src/iOS.Core/Utilities/iOSHelpers.cs

```diff
@@ -7,7 +9,37 @@ namespace Bit.iOS.Core.Utilities
 {
     public static class iOSHelpers
     {
-        public static System.nfloat? GetAccessibleFont<T>(double size)
+        [DllImport(ObjCRuntime.Constants.SystemLibrary)]
+        internal static extern int sysctlbyname([MarshalAs(UnmanagedType.LPStr)] string property, IntPtr output,
+            IntPtr oldLen, IntPtr newp, uint newLen);
+
+        // Returns the difference between when the system was booted and now in seconds, resulting in a duration that
+        // includes sleep time.
+        // ref: https://forums.xamarin.com/discussion/20006/access-to-sysctl-h
+        // ref: https://github.com/XLabs/Xamarin-Forms-Labs/blob/master/src/Platform/XLabs.Platform.iOS/Device/AppleDevice.cs
+        public static long? GetSystemUpTimeSeconds()
+        {
+            long? uptime = null;
+            try
+            {
+                var property = "kern.boottime";
+                var pLen = Marshal.AllocHGlobal(sizeof(int));
+                sysctlbyname(property, IntPtr.Zero, pLen, IntPtr.Zero, 0);
+                var length = Marshal.ReadInt32(pLen);
+                var pStr = Marshal.AllocHGlobal(length);
+                sysctlbyname(property, pStr, pLen, IntPtr.Zero, 0);
+                var timeVal = Marshal.PtrToStructure<TimeVal>(pStr);
+                var now = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
+                if (timeVal.sec > 0 && now > 0)
+                {
+                    uptime = now - timeVal.sec;
+                }
+            }
+            catch { }
```

Comment:
Missing `finally { ... }` block to properly deallocate the memory being allocated using `Marshal.AllocHGlobal()`, otherwise this would cause a memory leak issue. Use `Marshal.FreeHGlobal(n)`

neat example can be found here: https://stackoverflow.com/a/17562399